### ISSUE TITLE
Apache issues

### ISSUE THREAD

**Issue by franciscop** [bmullan91/express-subdomain#16]
When I try using it by adding the specified lines to `/etc/hosts`, I run into a problem. The apache page appears and says " Apache2 Ubuntu Default Page". I've tried doing `local.dev`, `local.com`, `www.local.com`, etc with no avail. My current one is:

```
127.0.0.1	localhost
127.0.0.1	local.com
127.0.0.1	www.local.com
127.0.0.1	es.local.com
```

I've also trying appending the port in the browser such as `local.com:3000` or in the `/etc/hosts` but all of them either show the Apache page or don't load at all. Any idea of how to solve this?

**Comment by bmullan91** [comment]
Hi @franciscop, node doesn't use apache. A web server in node is managed by the `http` module and created by `http.createServer` - express manages this for us generally though.

To check that everything is ok try this:

1. I'd stop apache via `service httpd stop`
2. `cd project` & `npm i express-subdomain`
3. `cd node_modules/express-subdomain`
4. `npm test`

**Comment by franciscop** [comment]
I know that node doesn't use apache, what I mean is that I got redirected there even following the steps. However stopping apache with `service apache2 stop` fixed the redirection to apache.

After doing 1-3 and then `npm install --dev` (for installing mocha locally), I could run npm test. It said `  24 passing (134ms)`, but before and after the tests it repeated several times `sudo: unable to resolve host Francisco`.

This issue is solved, thank you. However I found a new and strange one in Firefox, I'll post it as new issue since it's solved and might be a good future reference.
